cypress/components/action-popover/action-popover.cy.tsx

```diff
@@ -743,6 +754,7 @@ context("Test for ActionPopover component", () => {
       CypressMountWithProviders(<ActionPopoverNestedInDialog />);
 
       actionPopoverButton().eq(0).click();
+      // eslint-disable-next-line jest/valid-expect-in-promise
```

Comment:
why do we need this?

Reply:
This was because it didn't like the `then...` so I applied the Quick Fix and these disable next lines were added. I've removed them now and rewritten it without the `then`. Although I was worried there'd be a timing issue, these tests seem to work correctly now.